qemu/tests/cfg/win_vioser_test.cfg

```diff
@@ -0,0 +1,24 @@
+- win_vioser_test:
+    type = win_vioser_test
+    kill_vm = yes
+    only Windows
+    cdroms += " virtio"
+    driver_name = vioser
+    virtio_ports = "com.redhat.rhevm.vdsm1"
+    virtio_port_type = serialport
+    data = "hello world"
+    variants:
+        - with_vioser_test:
```

Comment:
This looks very similar to the base name, how about use `@default` here?

Reply:
vioser-test.exe and benchmark.exe are two different tools. I'd like to specify detailed name in the variants.
How about modify the cfg and .py name to win_serial_tool_test?

Reply:
> vioser-test.exe and benchmark.exe are two different tools. I'd like to specify detailed name in the variants.
> How about modify the cfg and .py name to win_serial_tool_test?

Hm, sounds better.

Reply:
modified, thanks